src/content/docs/faq/supporter.md

```diff
@@ -15,6 +15,8 @@ You will receive a colored prefix to indicate your supporter tier.
 
 ![](../../../assets/supporter/prefixes.png)
 
+Your tier is decided by the *total* amount of money you have ever donated to ReconnectedCC. From €25, you gain tier 2, and from €75 you gain tier 3.
```

Comment:
I wonder if a cool graph or table could be put here with the icons of each tier, but I'm not gonna hold my breath on that, it's a non-issue